CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 5.1.1
+
+BREAKING CHANGES:
+
+* The support_physical_cluster_replication flag has been removed from
```

Comment:
nit: I think we should write this as "... SupportPhysicalClusterReplication field...". Users of this SDK don't see the proto field name, just the struct field.